src/migrations/20191218075931-create-users.js

```diff
@@ -64,9 +64,6 @@ export function up(queryInterface, Sequelize) {
       type: Sequelize.BOOLEAN,
       defaultValue: false
     },
-    lineManagerId: {
```

Comment:
Why is this column removed from migration?

Reply:
There were duplicates from develop. both in migration and model